src/lib/nvme_pt_ioctl.c

```diff
@@ -2497,3 +2500,25 @@ int opal_write_shadow_mbr_pt(struct sed_device *dev, const struct sed_key *key,
 	return ret;
 }
 
+int opal_block_sid_pt(struct sed_device *dev, bool hw_reset)
+{
+	struct opal_device *device = dev->priv;
+	int ret;
+
+	/* Send Block SID authentication command, no IF_RECV response is expected */
```

Comment:
why to have 2 comments one after another. Can you please merge it? e.g.
/* Send Block SID authentication command, no IF_RECV response is expected
 * Set Hardware Reset in LSB of the first byte in
 * BlockSID payload based on Clear Event flag user
 * supplied through hwreset argument */

Reply:
Done change